provisioner/terraform/provision_test.go

```diff
@@ -703,13 +705,57 @@ func TestProvision(t *testing.T) {
 				}},
 			},
 		},
+		{
+			Name:       "workspace-owner-login-type",
+			SkipReason: "field will be added in provider version 1.1.0",
```

Comment:
note: I'm getting the ducks in a row now so that when the provider is released we can lock this in

Reply:
Nit: I wonder if we could expand this to be more strictly version-dependent?
For example, the tests could have a defined version under which they work and if that version does not exist then the test can be skipped.

Reply:
That would be wonderful to have!